PWGMM/Lumi/Tasks/fitLumi.cxx

```diff
@@ -0,0 +1,116 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Common/DataModel/EventSelection.h"
+#include "DataFormatsFDD/Digit.h"
+#include "DataFormatsFIT/Triggers.h"
+#include "Common/DataModel/FT0Corrected.h"
+
+#include "CCDB/CcdbApi.h"
+#include "CommonDataFormat/BunchFilling.h"
+#include "CCDB/BasicCCDBManager.h"
+#include "DataFormatsParameters/GRPObject.h"
+#include "DataFormatsParameters/GRPLHCIFData.h"
+#include "TH1F.h"
+#include "TH2F.h"
+#include <iostream>
+
+// author: arvind.khuntia@cern.ch
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+using BCPattern = std::bitset<o2::constants::lhc::LHCMaxBunches>;
+const int nBCsPerOrbit = o2::constants::lhc::LHCMaxBunches;
+
+#define FIND_INDEX(vector, element) (std::find((vector)->begin(), (vector)->end(), (element)) != (vector)->end() ? std::distance((vector)->begin(), std::find((vector)->begin(), (vector)->end(), (element))) : -1)
+
+struct fitLumi {
+  int nTF = 0;
+  HistogramRegistry registry;
+  int nBins, nCollBins;
+  int relTS;
+  Configurable<ULong64_t> startTimeInS{"startTime", 1668079200, "unix start time"};
```

Comment:
These three configurables look like they could be taken (or calculated) from CCDB information, is it not the case?

Reply:
I know it, will be in the next version. Thanks.